tools/pip/doc/CU101_ADDITIONAL.md

```diff
@@ -21,7 +21,7 @@ This package supports Linux and Windows platforms. You may also want to check:
 - [mxnet-cu112](https://pypi.python.org/pypi/mxnet-cu112/) with CUDA-11.2 support.
```

Comment:
can we add a newline here between the text and the list?

Reply:
Added new lines in all variant files. Thanks for the suggestion!